styles/blocks/address.scss

```diff
@@ -0,0 +1,18 @@
+.address {
+
+  &__label {
+    color: #9FADC2;
+    font-size: 14px;
+    font-family: Inter;
+    line-height: 140%;
+  }
+
+  &__content {
+    color: #3E3E3E;
+    font-size: 16px;
+    font-family: Inter;
+    line-height: 140%;
+
+    padding: 8px 0 24px;
+  }
+}
```

Comment:
It's a good practice to add a new line at the end of the file. This is a POSIX standard and helps with file concatenation. **Please add it to all your .scss files**

Reply:
Thank you for all suggestions. I have made changes. Pleae make review now :)